File: Variables Operaciones/1_08.FacturacionVb/Facturacion.vb
Imports System
'1.8. Crea un proyecto llamado “Facturación” en el cual se ingresa el nombre de un artículo y su
'costo.La utilidad es del 150 por ciento y el impuesto es el 21 por ciento. Calcular e imprimir
'artículo, utilidad, impuesto y precio de venta.
Module Facturacion
    Sub Main(args As String())
        Dim articulo As String = nombreArticulo()
        Dim precio As Single = costoArticulo()
        Dim precioUtilidad As Single = precio * 150 / 100
        Dim iva As Single = (precio + precioUtilidad) * 0.21
        Console.WriteLine("Artículo : {0}" & vbCrLf &
                          "Precio: {1}" & vbCrLf &
                          "Utilidad: {2}" & vbCrLf &
                          "Impuesto : {3}" & vbCrLf &
                          "PRECIO VENTA: {4}", articulo, precio, precioUtilidad, iva, precio + precioUtilidad + iva)
    End Sub

    Function nombreArticulo() As String
        Console.Write("Ingrese nombre de un producto : ")
        Return Console.ReadLine
    End Function

    Function costoArticulo() As Single
        Console.Write("Ingrese costo del articulo : ")
        Return Console.ReadLine
    End Function
End Module

File: Variables Operaciones/1_08.FacturacionCs/Facturacion.cs
using System;
/*1.8. Crea un proyecto llamado “Facturación” en el cual se ingresa el nombre de un artículo y su
costo. La utilidad es del 150 por ciento y el impuesto es el 21 por ciento. Calcular e imprimir
artículo, utilidad, impuesto y precio de venta.*/
namespace _1_08.FacturacionCs
{
    class Facturacion
    {
        static void Main(string[] args)
        {
            String articulo = nombreArticulo();
            Single precio = costoArticulo();
            Single precioUtilidad = precio * 150 / 100;
            Single iva  = (precio + precioUtilidad) * 21 /100;
            Console.WriteLine("Artículo : {0}" +
                          "Precio: {1}" +
                          "Utilidad: {2}" +
                          "Impuesto : {3}" +
                          "PRECIO VENTA: {4}", articulo, precio, precioUtilidad, iva, precio + precioUtilidad + iva);

            string nombreArticulo() {
                Console.Write("Ingrese nombre de un producto : ");
                return Console.ReadLine();
            }
            Single costoArticulo() {
                Console.Write("Ingrese costo del articulo : ");
                return Convert.ToSingle(Console.ReadLine());
            }
        }
    }
}
